### ISSUE TITLE
find a way around changelog conflicts

### ISSUE THREAD

**Issue by limlam96** [lvgig/tubular#325]
These are becoming a pain, some suggestions below:

https://medium.com/@nettsundere/on-reducing-changelog-merge-conflicts-1eb23552630b

From this article, I potentially lean towards just adding placeholder lines to the file?

**Comment by adamsardar** [comment]
Would #254 be a potential solution? Its come up in several places - internal and external, so might be worth my hammering out a poc?

**Comment by limlam96** [comment]
yeah sounds better - maybe a question as to whether while we're prioritising polars stuff we want to implement the quick/rough solution of just adding placeholder lines as a stop gap though

**Comment by marwin1991** [comment]
I don't know if you are looking for such a  tool, but you can check out https://logchange.dev/tools/logchange/getting-started/

Jinja templates should meet your needs.

Apache Solr using it exactly because of wasting time for pull requests conflicts

https://lists.apache.org/thread/4dotf4qx4ss3qr3xonv2y63v7wdg40nt
